Task2BS/ExperimentViewController.h

```diff
@@ -0,0 +1,13 @@
+//
+//  ExperimentViewController.h
+//  Task2BS
+//
+//  Created by oded regev on 11/25/17.
+//  Copyright © 2017 Gai Carmi. All rights reserved.
+//
+
+#import <UIKit/UIKit.h>
+
+@interface ExperimentViewController : UIViewController
```

Comment:
`ExperimentViewController` will be the view controller which holds the `UIFloatLabelTextField`